ci/deps/travis-35.yaml

```diff
@@ -19,33 +18,40 @@ dependencies:
   - numexpr
   - numpy=1.13*
   - openpyxl=2.4.0
+  - pathlib
   - patsy
   - psycopg2
   - py
   - pyarrow=0.9.0
   - PyCrypto
-  - pymysql=0.6.3
+  - pymysql=0.6.6
   - pytables
   - blosc=1.14.3
   - python-blosc
   - python-dateutil=2.5.0
-  - python=2.7*
-  - pytz=2013b
-  - s3fs
+  - python=3.5*
+  - pytz=2015.4
   - scipy
-  - sqlalchemy=0.9.6
+  - sqlalchemy=1.0.8
+  - s3fs
   - xarray=0.9.6
   - xlrd=1.0.0
-  - xlsxwriter=0.5.2
-  - xlwt=0.7.5
+  - xlsxwriter=0.7.7
+  - xlwt=1.0.0
   # universal
-  - pytest>=4.0.2
   - pytest-xdist
   - pytest-mock
-  - moto==1.3.4
+  - moto
   - hypothesis>=3.58.0
   - isort
+  # satisfy fastparquet deps with conda
```

Comment:
these are needed explicity?

Reply:
i would rather remove fastparquet from this build then, we have it installed for 1 or more of 3.6/3.7 already I think (pls check)

Reply:
Removed. Remaining jobs with `fastparquet`:
```
ci/deps\azure-windows-36.yaml:9:  - fastparquet>=0.2.1
ci/deps\travis-36-doc.yaml:9:  - fastparquet>=0.2.1
ci/deps\travis-36.yaml:10:  - fastparquet>=0.2.1
```